channels/rdpsnd/server/rdpsnd_main.c

```diff
@@ -59,6 +59,9 @@ static UINT rdpsnd_server_send_formats(RdpsndServerContext* context)
 	BOOL status = FALSE;
 	ULONG written = 0;
 
+	if (context->num_server_formats == 0)
```

Comment:
looks like a bug.
1. `PCM` should always be supported 
2. the channel is *required* for `RDP` protocol to properly work (don't ask why, did not design it)
3. if not the whole channel should not be started instead of aborting setup sequence here